potassium/potassium.py

```diff
@@ -199,7 +203,15 @@ def handle(path):
                 abort(404)
 
             endpoint = self._endpoints[route]
-            return self._handle_generic(route, endpoint, request)
+            return self._handle_generic(endpoint, request)
+
+        # add __status__ endpoint
+        @flask_app.route('/__status__', methods=["GET"])
```

Comment:
I think it's good to communicate elsewhere, that if you call this when the app is deployed on banana, you'll get billed for GPU time.

Reply:
discussed offline but i think this is ok given we only forward POSTs through proxy so even if a user tries to use this they won't get billed.